doc/make.py

```diff
@@ -309,5 +309,5 @@ def build_all():
                               ' are %s' % (command, funcd.keys())))
         func()
 else:
-    all()
+    raise SystemExit("Missing target.")
```

Comment:
"You ran ./make.py instead of ./make.py html or ./make.py pdf or ./make.py $whatevertarget" (cf https://grox.net/doc/unix/unix_shell_humor.html).